src/rp2_common/pico_stdio_rtt/stdio_rtt.c

```diff
@@ -26,11 +26,11 @@ void stdio_rtt_deinit(void) {
 }
 
 static void stdio_rtt_out_chars(const char *buf, int length) {
-    SEGGER_RTT_Write(0, buf, length);
+    SEGGER_RTT_Write(0, buf, (unsigned)length);
 }
```

Comment:
Presumably `length` can never be negative (?), so perhaps it makes sense to just change the function-prototype to `stdio_rtt_out_chars(const char *buf, unsigned int length)` ?
(Apologies if this is a stupid suggestion)

Reply:
Unfortunately the prototype of this function is defined by stdio_driver_t 

```
stdio_driver_t stdio_rtt = {
    .out_chars = stdio_rtt_out_chars,
```

Reply:
Ahhh, so it _was_ a stupid suggestion :laughing:  Fair enough.